manifests/named_instance.pp

```diff
@@ -157,6 +164,14 @@
     ]
   }
 
+  @@nebula::profile::proxied_app { $title:
```

Comment:
I think you'll want to keep this as-is, since some of these parameters are also used elsewhere.